src/discrete-layer/components/map-container/layers-footprints.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useEffect, useState } from 'react';
+import { observer } from 'mobx-react-lite';
+import { CesiumGeojsonLayer, CesiumColor } from '@map-colonies/react-components';
+import { Feature, FeatureCollection, Geometry, Polygon } from 'geojson';
+import polygonToLine from '@turf/polygon-to-line';
+import bbox from '@turf/bbox';
+import bboxPolygon from '@turf/bbox-polygon';
+import { useStore } from '../../models/rootStore';
+
+export const LayersFootprints: React.FC = observer(() => {
+  const { discreteLayersStore } = useStore();
+  const [layersFootprints, setlayersFootprints] = useState<FeatureCollection>();
+
+  // REMARK: Layers footprint boundingboxes  
+  useEffect(() => {
+    if (discreteLayersStore.layersImages) {
+      const footprintsCollection: FeatureCollection = {
+        type: 'FeatureCollection',
+        features: []
+      }
+      const footPrintsFeaturesArray = discreteLayersStore.layersImages.map((layer) => {
+        let geometry: Geometry = layer.geojson as Geometry;
+        switch(layer.geojson?.type){
+          case 'Polygon':
+            geometry = (polygonToLine(geometry as Polygon) as Feature).geometry;
+            break;
+          case 'MultiPolygon':
+            //get bbox of feture, then convert it to polygon and finally get linestring of polygon
+            geometry = (polygonToLine(bboxPolygon(bbox(geometry))) as Feature).geometry;
+            break;
+          default:
+            break;
+        }
+        
+        const footPrint: Feature = {
+          type: 'Feature',
+          geometry: { 
+            ...geometry,
+          },
+          properties: {
+            name: layer.name,
+            description: layer.description,
+          },
+        }
+        return footPrint;
+      });
+      footprintsCollection.features.push(...footPrintsFeaturesArray);
+      setlayersFootprints(footprintsCollection);
+    }
+  }, [discreteLayersStore.layersImages]);
+
+  // REMARK: Layers feature footprint as is
```

Comment:
remove REMARK?

Reply:
It's a second **geojson as is** implementation

Remained unchanged